### PR TITLE
GH-1991 Invalid graph event params

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-1991

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage Please Review (may have implications outside of HipieDDL)

**Comment by jbrundage** [comment]
@GordonSmith This looks good to me, but I'll have to refactor the AVCC implementation of graph_Graph

File: src/graph/Graph.js [hunk 1]
```diff
@@ -501,10 +501,20 @@
                 context._selection.click(d, d3.event);
             })
             .on("click", function (d) {
-                context.vertex_click(d, d3.event);
+                var vertexElement = d3.select(this).select(".graph_Vertex");
+                var selected = false;
+                if (!vertexElement.empty()) {
+                    selected = vertexElement.classed("selected");
+                }
+                context.vertex_click(d.data(), "", selected);
```
Comment (GordonSmith) [inline src/graph/Graph.js]: @jbrundage I think I should change this to:
```
context.vertex_click(d.data(), "", selected, {
   graphVertex:  d 
});
```
You will still need a refactor, but it should be a small one?

**Comment by GordonSmith** [comment]
@jbrundage I updated this PR so the event now includes the vertex/edge in a "more" param:  
```
vertex_click = function (row, col, sel, more) { 
    if (more && more.vertex) { 
        console.log("Vertex Click: " + more.vertex.id()); 
    } 
}; 
```
type of thing - please re-review

**Comment by jchambers-ln** [comment]
Looks good to me but I'll defer to @jbrundage since he has been reviewing this already

**Comment by GordonSmith** [comment]
Its more so @jbrundage knows to change his code!

**Pull request merged**